### ISSUE TITLE
How to reproduce evaluation result to kitti dataset

### ISSUE THREAD

**Issue by shuiyuejihua** [DepthAnything/Depth-Anything-V2#289]
i use the kitti dataset 

[Download manually selected validation and test data sets (2 GB)](https://s3.eu-central-1.amazonaws.com/avg-kitti/data_depth_selection.zip)

use Depth-Anything-V2-Large checkpoint

**post preprocess**

kitti dataset gourdtruth png :  
    gourdtruth_depth = read image / 256.

depth = model.infer(img) , just like run.py,  contains image preprocess

finally, evalate  the depth, gourdtruth_depth 

valid mask is target>0 

```
def eval_depth(pred, target):
    assert pred.shape == target.shape
    diff = pred - target
    abs_rel = np.mean(np.abs(diff) / target)
    thresh = np.maximum(target / pred, pred / target)
    d1 = (thresh < 1.25).mean()
    return abs_rel, d1
```

result, the absrel :30 , delta1 ：0.016
and i find that if use the pred /80, the result is better ： absrel 0.6， delta1 0.11

but the table 2

<img width="1056" height="406" alt="Image" src="https://github.com/user-attachments/assets/6f59b4df-79f9-43e6-93f3-a176692581f5" />

is there anything tips？

**Comment by GreaBugs** [comment]
兄弟我也在复现这个，你复现出来没？可以交流一下

**Comment by Kai-Yin-UoA** [comment]
I just find the point. This is their result on relative MDE, which means they got this result by validate with the given pretrain model. The metric MDe result need to be fine-tuned on other datasets, such as vkitti and hypersim. And the result of fine-tuned metric depth is little bit lower than this result (All labeled data abs rel: 0.081 d1: 0.937). The author just released the training set list of vkitti and hypersim. If you also want to reproduce this result, you have to prepare the other datasets as well and ask author for the training/val set lists.

**Comment by fogradio** [comment]
> [#280](https://github.com/DepthAnything/Depth-Anything-V2/issues/280)
> 
> [#157](https://github.com/DepthAnything/Depth-Anything-V2/issues/157)
> 
> I use this：
> 
> ```
> inverse_truth = 1.0 / (truth_masked + epsilon)
> A = np.concatenate([output_masked, np.ones_like(output_masked)], axis=-1)
> scale, shift = np.linalg.lstsq(A, inverse_truth, rcond=None)[0]
> alignmented_output = 1.0 / (output_map * scale + shift + epsilon)
> ```
> 
> kitti-depth dataset [Download manually selected validation and test data sets (2 GB)](https://s3.eu-central-1.amazonaws.com/avg-kitti/data_depth_selection.zip)
> 
> absrel 0.081， delta1 0.946

But doing such an operation puts it in the realm of **relative depth (affine-invariant)** evaluation, not **metric depth (absolute, in meters)** evaluation. Metric depth evaluation prohibits fitting a per-image scale/shift; it requires using the model’s output in “meters” directly to compute the metrics.
